### PR TITLE
Network troubleshooting using traceroute

### PR DESCRIPTION
**Pull request by Su-sid**
### Article checklist
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [+ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Hello @Su-sid 👋 
Congratulation on your first article 👏 and we are happy to have you in our amazing community 👀
Please do the following:
- Change the conclusion, maybe make a *Further reading* section. Look at the previous articles to get a feel for the type of content we are looking for.

- I noticed the hero has low image quality. We are working on a new [hero images template](https://github.com/ahmadmardeni1/Canva). Take a look, it may help you with creating a better one.

- The windows traceroute image example has a very low image quality (image number six) with white edges. Change it. You can use this image, which is an example of using traceroute with google DNS in windows.

Capture

- Make sure you go through the changes I made here cd201d6 so that you can prevent them in your future articles.

Please ping me when you are done so that we can start the tech review.

File: articles/network-troubleshooting-using-traceroute/index.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+> **Note:** Understanding what packets are, allows us to understand better how the traceroute utility functions. 
+
+Data packets are units of communication that travel along a given network path [site]. Any data packet sent over the internet contains the Time To Live (TTL) field located in their headers. The TTL field is essential in a data packet as it prevents data packets from forever traveling in an endless route while searching for their destination. Traceroute uses the TTL field to measure the distances between the source and destination and find the devices/routers between the paths.
+
+When forwarding a packet, routers edit the TTL field of packets by decreasing the packet's TTL value by one every time the packet reaches a device on the path. When the TTL value of a packet is zero, the router discards the packet and sends back a 'TTL exceeded message.' to the source.
+
+The TTL exceeded message is what traceroute utility looks for as it contains information, like:
+
+- The time it took to reach a hop referred to as the round trip time(RTT) 
+- The identity, given by the name or IP address, of the device/server/router, replying
```
[the remark this answers is not in the document]
Comment (Su-sid) [reply articles/network-troubleshooting-using-traceroute/index.md:24]: yes.

[the remark this answers is not in the document]
Comment (Su-sid) [reply articles/network-troubleshooting-using-traceroute/index.md:24]: I have made the necessary correction by removing the last comma and added a full stop.

**Comment by Su-sid** [comment]
@ahmadmardeni1  done, please check.

**Comment by Su-sid** [comment]
@ahmadmardeni1 kind remove the "needs author page" tag as there is an authors page available under the folder "sudi-david"

**Comment by Su-sid** [comment]
@ahmadmardeni1 kindly remove the "needs author page" tag as there is an authors page available under the folder "sudi-david"

**Comment by ahmadmardeni1** [comment]
@Su-sid This label is for us. To create an author page for you on the website.

**Comment by Su-sid** [comment]
@ahmadmardeni1 Thank you for clarifying
.

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by hectorkambow** [comment]
Great work gentlmen - some minor edits I made during the final review. 

@Su-sid  please be sure to go over them - to incorporate into future articles where applicable. 

cc @ahmadmardeni1  please do the same. 

Also - for articles that do not have front matter please be sure to add in by following [these instructions](https://github.com/section-engineering-education/engineering-education/blob/master/peer_reviewers/front-matter-process.md).

**Pull request merged**